src/gpl/src/nesterovPlace.cpp

```diff
@@ -241,6 +243,13 @@ void NesterovPlace::init() {
 
   debugPrint(log_, GPL, "replace", 3, "npinit: InitialStepLength {:g}", stepLength_);
 
+  if( pb_->insts().size() < 500 && isnan(stepLength_) ) {
+    npVars_.initialPrevCoordiUpdateCoef *= 10;
+    log_->warn(GPL, 321, "steplength = 0 detected. Rerun Nesterov::init() with initSLPCoef:{:g}",
```

Comment:
If the software is doing the action, use the gerund to avoid confusion. e.g. "Rerun Nesterov::init()" -> "Rerunning Nesterov::init()". If the user needs to do something, then it is okay to use the command form e.g. "Please rerun Nesterov::init()".

Reply:
I changed it as debugPrint